src/classes/BGE_DataImportBatchEntry_CTRL.cls

```diff
@@ -394,6 +396,22 @@ public with sharing class BGE_DataImportBatchEntry_CTRL {
         return [SELECT Count() FROM DataImport__c WHERE NPSP_Data_Import_Batch__c =: recordId];
     }
 
+    /*******************************************************************************************************
+    * @description Gets the Gift Entry defined list of Batch Table Columns as JSON.
+    * @param recordId the ID of the NPSP_Data_Import_Batch__c
+    * @return String: List of Batch Table Columns as JSON
+    */
+    private static String[] getGiftEntryDefinedBatchTableColumns(Id recordId) {
```

Comment:
The JS file of the GeGiftEntryFormApp [uses getRecord](https://github.com/SalesforceFoundation/NPSP/blob/46d9296bfb752e19fcf9fda42e657dc522190047/src/lwc/geGiftEntryFormApp/geGiftEntryFormApp.js#L578) to import some fields on the batch already.  Could the App use that to retrieve the value of the new Batch Table Columns field and pass it down to the table component?  Then this method wouldn't be needed and the DataImportModel wouldn't need to be altered.